app/controllers/ArrivalMessagesController.scala

```diff
@@ -38,7 +38,7 @@ import play.api.mvc.AnyContent
 import play.api.mvc.ControllerComponents
 import uk.gov.hmrc.http.HttpErrorFunctions
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
-import utils.CallOps._
+import v2.utils.CallOps._
```

Comment:
should use utils.CallOps._ for v1

Reply:
@sonisn1988 I moved it v2 because we now use it in HateosResponse rather than using the if statement 
https://github.com/hmrc/common-transit-convention-traders/pull/332/files#diff-c531ed67eb289711a1e6fa6155247b9095162c1e78f0b49fcf37169f2d43035dL36

When we delete V1 components we will want this to remain available